content/en/docs/tasks/security/authz-http/index.md

```diff
@@ -32,178 +30,250 @@ the product page, you can see the following sections:
-is accessible by services in the same namespace (i.e., `default`) and services in the `istio-system` namespace.
-
-Run the following command to create a namespace-level access control policy:
-
-{{< text_import file="examples/TestAuthorizationForHTTPServices.txt" snippet="enforcing_namespace_level_access_control_apply.sh" >}}
+## Enforce mesh-level access control
 
-Once applied, the policy has the following effects:
+Using Istio, you can easily setup mesh-level access control for all workloads in your mesh.
 
-*   Creates a `ServiceRole` `service-viewer` which allows read access to any service in the `default` namespace that has
-the `app` label
-set to one of the values `productpage`, `details`, `reviews`, or `ratings`. Note that there is a
-constraint specifying that
-the services must have one of the listed `app` labels.
+Run the following command to create a mesh-level access control policy `deny-all` policy
+in the root namespace, A policy in the root namespace will select all workloads in the mesh.
 
-    {{< text_import file="examples/TestAuthorizationForHTTPServices.txt" snippet="enforcing_namespace_level_access_control_service_viewer.yaml" >}}
+The root namespace is configurable in the [`MeshConfig`](/docs/reference/config/istio.mesh.v1alpha1/#MeshConfig)
+and has the default value of `istio-system`. If you have changed it to a different value from the
+default `istio-system`, Please update the value accordingly in the following examples.
 
-*   Creates a `ServiceRoleBinding` that assigns the `service-viewer` role to all services in the `istio-system` and `default` namespaces.
-
-    {{< text_import file="examples/TestAuthorizationForHTTPServices.txt" snippet="enforcing_namespace_level_access_control_bind_service_viewer.yaml" >}}
-
-You can expect to see output similar to the following:
-
-{{< text plain >}}
-servicerole "service-viewer" created
-servicerolebinding "bind-service-viewer" created
+{{< text bash >}}
+$ kubectl apply -f - <<EOF
+apiVersion: security.istio.io/v1beta1
+kind: AuthorizationPolicy
+metadata:
+  name: deny-all
+  namespace: istio-system
+spec:
+  {}
+EOF
 {{< /text >}}
 
-Now if you point your browser at Bookinfo's `productpage` (`http://$GATEWAY_URL/productpage`). You should see the "Bookinfo Sample" page,
-with the "Book Details" section in the lower left part and the "Book Reviews" section in the lower right part.
+The `{}` in the policy means `deny-all`, you can also set other mesh-level policy for your own needs.
 
-{{< tip >}}
-There may be some delays due to caching and other propagation overhead.
-{{< /tip >}}
+Point your browser at the Bookinfo `productpage` (`http://$GATEWAY_URL/productpage`).
+Now you should see `"RBAC: access denied"`. The error shows that the configured `deny-all`
+policy is working as intended, and Istio doesn't have any rules that allow any access to
+workloads in the mesh.
 
-### Cleanup namespace-level access control
+### Clean up mesh-level access control
```

Comment:
I think you should keep this policy, as it's more natural to show the namespace or workload policy effects later (granting access).
Again, I think we should change the narrative of this tutorial. Putting in 3 distinguished scopes is hard for user to understand why. Rather, you should showing deny-all via global, then open per-namespace and/or workload, pointing out the permission are always _additive_ in all cases.

Reply:
this policy is added again in the `Enforce workload-level access control` section where we explicitly talk about this process that grants more permission gradually. I was mainly trying to maintain the same structure but just switch to use the new policy in these tasks, I agree we probably need to rewrite the task completely, and I think it would be better to not to use bookinfo, but to use sleep/httpbin that gives us more flexibility to show different features of the API.